src/controllers/Controller.js

```diff
@@ -109,12 +132,12 @@ module.exports = class Controller {
         });
       }
 
-      if (user.verified === 'false') {
-        return response(res, {
-          data: { message: 'Email not verified. Please check your email for a verification link.' },
-          status: 401,
-        });
-      }
+      // if (user.verified === 'false') {
```

Comment:
Is there a reason to comment out verification?

Reply:
oh yeah that was for testing, that should be reverted.